services/management/ia/rules_service.go

```diff
@@ -92,8 +92,26 @@ type rule struct {
 	Annotations map[string]string   `yaml:"annotations,omitempty"`
 }
 
+// RemoveVMAlertRulesFiles removes all generated rules files (*.yml) on the ia path
+func (s *RulesService) RemoveVMAlertRulesFiles() error {
+	ruleFiles, err := filepath.Glob(rulesDir + "/*.yml")
+	if err == nil {
+
+		for _, file := range ruleFiles {
+			if err = os.RemoveAll(file); err != nil {
+				break
+			}
+		}
+
+		s.vmalert.RequestConfigurationUpdate()
+		s.alertManager.RequestConfigurationUpdate()
+	}
+
+	return err
+}
```

Comment:
```suggestion
func (s *RulesService) RemoveVMAlertRulesFiles() {
	matches, err := filepath.Glob(s.rulesPath + "/*.yml")
	if err != nil {
		s.l.Errorf("Failed to clean old alert rule files: %+v", err)
		return
	}

	for _, match := range matches {
		if err = os.RemoveAll(match); err != nil {
			s.l.Errorf("Failed to remove old rule file: %+v", err)
		}
	}
}
```


It can be just copy-pasted lines 127-137 from `WriteVMAlertRulesFiles`. Also we can use `RemoveVMAlertRulesFiles` inside `WriteVMAlertRulesFiles` for cleanup.